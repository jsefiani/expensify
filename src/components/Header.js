import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <nav>
            <NavLink to="/" activeClassName="is-active" exact={true}>
                Dashboard
            </NavLink>
            <NavLink to="/create" activeClassName="is-active">
                Create
            </NavLink>
        </nav>
    </header>
);

export default Header;