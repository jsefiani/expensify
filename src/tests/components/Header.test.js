import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from '../../components/Header';

test('Should render Header component correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot()
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});