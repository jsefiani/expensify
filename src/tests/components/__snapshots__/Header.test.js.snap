// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header component correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <nav>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/create"
    >
      Create
    </NavLink>
  </nav>
</header>
`;
